### PR TITLE
Making subscriptions work

### PR DESCRIPTION
**Pull request by itamarsch**

### CODE REVIEW

**Comment by itamarsch** [comment]
@giladwo you'll like this one

File: lib/net/hasura_helper.dart [hunk 1]
```diff
@@ -4,10 +4,21 @@ import "package:scouting_frontend/models/team_model.dart";
...
 
 GraphQLClient getClient() {
-  final HttpLink link = HttpLink(
+  final HttpLink httpLink = HttpLink(
     "https://orbitdb.hasura.app/v1/graphql",
   );
-  return GraphQLClient(link: link, cache: GraphQLCache());
+  final WebSocketLink webSocketLink = WebSocketLink(
+    "wss://orbitdb.hasura.app/v1/graphql",
+    config: SocketClientConfig(),
```
Comment (giladwo) [inline lib/net/hasura_helper.dart:12]: I can't recall using a `SocketClientConfig`, so perhaps that's what we were missing a few months ago

**Review by giladwo — APPROVED** [review]

**Comment by giladwo** [comment]
> @giladwo you'll like this one

Having web socket isn't the feature we need, so you'd better start converting all `Query`s to `Subscription`s :slightly_smiling_face: .

File: lib/net/hasura_helper.dart (hunk 1 above)
Comment (itamarsch) [reply lib/net/hasura_helper.dart:12]: This worked without it so it's something else maybe an update

**Pull request merged**